pkg/api/api.go

```diff
@@ -142,6 +147,10 @@ type CheckCodeStatusResponse struct {
 	// After this time the code will no longer be accepted and is eligible for deletion.
 	ExpiresAtTimestamp int64 `json:"expiresAtTimestamp"`
 
+	// LongExpiresAtTimestamp repesents the time when the long code expires, in
```

Comment:
is the "long code" different from the code? Still a little confused about how this differs from ExpiresAtTimestamp

Reply:
ExpiresAt is for the short code (the 8-12 digit one on-screen). Long.* is for the longer deep-link SMS codes that are sent to the user (and never displayed).

Reply:
ah. I think I got confused thinking 'Long' mean int64